### PR TITLE
ADD REMOVE INTERRACTION OF TO DO LIST

### PR DESCRIPTION
**Pull request by citec-47**
**### #ADD REMOVE INTERRACTION OF TODO LIST**
- **IMPLEMENTATIONS**
- deleting all items at ones
- deleting one item after the other
- can mark accomplish task
- list items are editable

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,125 @@
...
+window.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') addnewTask();
+});
+
+window.onload = () => {
+  mytasks = JSON.parse(localStorage.getItem('mytasks'));
+  if (mytasks) {
+    mytasks.forEach((fresh) => {
+      fresh.completed = false;
+    });
```
Comment (amejid) [inline src/index.js:101]: - [ ] You did a great job with the interactions. :+1: There is a minor issue here, when reloading the page tasks that were marked completed are now reverting to uncompleted tasks. That is not the behavior expected from the app. I think removing this `forEach` loop will help. ✍️

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,125 @@
...
-import { clear_my_Funct } from './modules/calling.check.js';
-
-const buttonAdd = document.getElementById('add-button');
-buttonAdd.addEventListener('click', btnAdd);
-const refresh = document.getElementById('refresh');
-refresh.addEventListener('click', refreshBtn);
-const clear = document.getElementById('clear');
-clear.addEventListener('click', clear_my_Funct);
-addHtml();
+/* eslint-disable  import/no-cycle, import/no-mutable-exports */
```
Comment (amejid) [inline src/index.js:1]: - [ ] Using linters can help developers write better code by enforcing consistent code styles, identifying common programming errors, and highlighting potential performance issues. Kindly remove the lines where linters are disabled. 🤝️ Please do not modify any of the linter configuration files when addressing the issue. 🥸️

File: src/modules/calling.update.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports, import/prefer-default-export */
```
Comment (amejid) [inline src/modules/calling.update.js:1]: - [ ] Same here.

File: src/modules/calling.task.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports, import/prefer-default-export */
```
Comment (amejid) [inline src/modules/calling.task.js:1]: - [ ] Same here.

File: src/modules/calling.delete.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports, import/prefer-default-export */
```
Comment (amejid) [inline src/modules/calling.delete.js:1]: - [ ] Same here.

File: src/modules/calling.check.js [hunk 6]
```diff
@@ -1,21 +1,26 @@
-/* eslint-disable camelcase */
-import storage from './call.storage.js';
-import main_re_Arr from './calling.const.js';
+/* eslint-disable  import/no-cycle */
```
Comment (amejid) [inline src/modules/calling.check.js:1]: - [ ] Same here.

File: src/modules/call.storage.js [hunk 7]
```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable camelcase */
-import main_re_Arr from './calling.const.js';
+/*eslint-disable*/
```
Comment (amejid) [inline src/modules/call.storage.js:1]: - [ ] Same here.

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @citec-47, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Clear completed functionality working as expected :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @citec-47 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**